homeassistant/components/qnap/sensor.py

```diff
@@ -360,6 +360,8 @@ def extra_state_attributes(self):
                 ATTR_MASK: data["mask"],
                 ATTR_MAC: data["mac"],
                 ATTR_MAX_SPEED: data["max_speed"],
+                ATTR_PACKETS_RX: data["rx_packets"],
+                ATTR_PACKETS_TX: data["tx_packets"],
```

Comment:
Why should these be state attributes and not separate states?

Reply:
I just followed the existing code, and as ATTR_PACKETS_ERR is a state attribute I added those there, too. Otherwise, ATTR_PACKETS_ERR should be a separate state, too.

Reply:
> I just followed the existing code

These are pretty old constructs. We avoid the use of attributes nowadays.